1-Authentication/1-sign-in/AppCreationScripts/Configure.ps1

```diff
@@ -0,0 +1,197 @@
+[CmdletBinding()]
+param(
```

Comment:
>param [](http://example.com/codeflow?start=0&length=5)

note that this script has issues with , in appsettings.json

Reply:
Will address this in codegenerator